src/NUnitFramework/framework/nunit.framework-netstandard13.project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
I think I see the problem. I am building on the command line and see no warnings, but I see them building in Visual Studio. I just did a `which dotnet` on the command line and it looks like Cake overrides the PATH and puts the version of dotnet it downloads in, so I end up using the older version at the command line, not the version specified in `global.json`.